### ISSUE TITLE
death attack on multiple networks

### ISSUE THREAD

**Issue by Romko666** [martin-olivier/airgorah#132]
Tell me please, is this normal behavior of the program?
You press death, select one client or all, the attack happens perfectly!
But when you press the button (near the channel selection) _Hop on all channel_... the attack stops.
And continues only after pressing the button _Focus the channel_...
I can't deauth on several networks at once?

**Comment by martin-olivier** [comment]
Hello @Romko666,

If you are performing a deauth attack, let's say on channel 6, and after that, click on `Hop on all channel...` button, the `aireplay-ng` process will almost not be able to send deauth packets since `airodump-ng` will change the channel very often.

I will make some fixes to avoid this scenario

**Comment by martin-olivier** [comment]
Hello @Romko666,

The channel filter option is doing the job your are describing, by only focusing the channel of the access point you are interested in.

I do not see a usecase for BSSID filtering

**Comment by Romko666** [comment]
Yes, it will only search for all available access points on this channel.
The filter will allow you to select only one/two as needed.

Userfriendly))

**Comment by Romko666** [comment]
@martin-olivier Friend, has the project been cancelled?

**Comment by martin-olivier** [comment]
Hello @Romko666,

No, do not worry, this project is not cancelled, I was just not able to work on it the last 6 months because I had a lot of work for both my studies and my job.

I had my last studies exam yesterday so from now, I will be able to continue the development of this project.

**Comment by Romko666** [comment]
@martin-olivier Congratulations on graduating! ;)

**Comment by martin-olivier** [comment]
Hello @Romko666,

Sorry for the long delay, I just pushed the fix, this will be available in the next release (0.8.2) but you can already try it by building the project from source
